application-workflow-detailed.md

```diff
@@ -0,0 +1,442 @@
+# Detailed Application Authentication
+
+This document outlines, in detail, the login and request process for an application using WebId-OIDC. In general, our user, *Alice* will be using a thrid-party application at `https://www.decentphotos.example`  to access data on both her pod at `https://alice.example` and her friend, Bob's pod at `https://bob.example`.
+
+## Actors
+
+In this example a multitude of actors are at play:
+
+**Alice** - Alice will be providing consent for decentphotos to use her pod. Let's assume that Alice is using a standard web browser.
+
+**Bob's Pod (RS)** - We will be trying to access photos Bob's Pod, known in the OIDC world as a Resource Server (RS). Bob is a friend of Alice. For this use case, let's assume that Bob has previously indicated via access control that Alice may access his photo using any app. You can read more about access control [here](https://github.com/solid/web-access-control-spec#referring-to-origins-ie-web-apps). For this example, bob's pod is at `bob.solid.example`.
```

Comment:
typo - should be "trying to access photos stored in Bob's Pod"

Reply:
For clarification, I changed all references to "app" to "webapp"